swing-lib/src/main/java/org/triplea/swing/SwingComponents.java

```diff
@@ -139,11 +139,13 @@ public void actionPerformed(final ActionEvent e) {
             });
   }
 
-  public static JTabbedPane newJTabbedPane() {
-    return newJTabbedPane(900, 600);
+  public static JTabbedPane newJTabbedPane(final int width, final int height) {
+    final JTabbedPane tabbedPane = new JTabbedPaneWithFixedWidthTabs();
```

Comment:
Was this line supposed to be `newJTabbedPane()`?
I'm a bit confused as I thought we were removing fixed width tabs, and they seem to still be around. Wondering what I'm missing.

Reply:
Whoops, yes. Fixed.